### DISCUSSION TITLE
Mangehaftes Regelungsverhalten des DPL mit HM-600

### DISCUSSION THREAD

**Discussion by hoschiking** [hoylabs/OpenDTU-OnBattery#1009] [Q&A]
Ich habe gestern den seit Monaten gut laufenden HM-300 durch einen HM-600 ersetzt, um die
max. Einspeiseleistung zu verdoppeln.
Es funktioniert soweit ganz gut, bis auf das Regelungsverhalten des DPL, das ist wesentlich schlechter als vorher.
Die Kabelage hab ich überprüft, alles top.
Hier die vorletzte Nacht mit dem HM-300
![2024-05-29 08_45_32-Window](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/159241718/7de95fe3-6904-4096-854a-95e34de2f2a9)

Hier die letzte Nacht mit dem HM-600
![2024-05-29 08_46_07-Window](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/159241718/05be1592-5765-49f8-9d36-2574f8724597)

Die Einstellungen des DPL sind identisch, bis auf die nötigen Änderungen durch den Tausch.
![2024-05-29 09_30_39-Window](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/159241718/13c979f3-050e-4778-979b-6f9b80a9fb3f)

Das Ding schwingt nicht nur permanent, sondern erreicht selten den angestrebten Netzbezug von 0 W.
Meistens bleibt es im positiven Bereich hängen.
Einer eine Idee?

**Comment by schlimmchen** [comment]
Dein minimales Leistungslimit ist zu gering. Du musst es ungefähr verdoppeln. Dass die Hoymiles WR schwingen, wenn sie sehr kleine Leistung erbringen sollen, haben wir schon mehrfach festgestellt.

Brauchst du denn überhaupt so kleine Leistung? Sieht mir eher so aus, als würde der WR auch bei 100W schwingen.

Hast du beide Anschlüsse des HM-600 angeschlossen, oder nur einen?

Was passiert, wenn du den DPL ausschaltest und eine feste Leistung einstellst? Bleibt der Output dann stabil?

Was hast du für ein Batteriesystem (Spannung)?

**Comment by hoschiking** [comment]
Ich werde es mal testen mit dem min Leistungslimit auf 20W, obwohl ich es schon gestern probiert habe, ohne Auswirkung.
Das Schwingen tritt auch bei höheren Leistungen auf, wie du schon selbst festgestellt hast.
Meine Grundlast beträgt in etwa 80-100 Watt und die schafft er die ganze Nacht nicht einmal auszugleichen.
Beide Anschlüße angeschlossen, 24 Volt System, daher die Begrenzung auf 400W. Bis 400W stellt sich die Leistung
sehr gut ein, wenn man es manuell einstellt. Dies habe ich natürlich direkt nach der Installation alles getestet und alles über
eine Stunde bei 400W laufen lassen.  
Auch die Ströme verteilen sich bis auf 0,1A über beide Strings gleichmässig ein. Alles schon direkt nach dem Anschließen getestet,
bin ja kein Neuling und ich kenne die Problematik der HM-600, insbesondere bei 24V Systemen. Alles berücksichtig.
Ein Screenshot bei DPL aus, P auf 400W manuell eingestellt. Da schwingt nichts, Spannungen sind stabil.
![2024-05-29 09_05_49-Window](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/159241718/48bdd5ff-9bdc-449f-b0d2-ce8f67190a84)

**Comment by TomBrett72** [comment]
Hallo zusammen,
hab auch nen HM-600 und 24V. In meinen Augen ist die Hysterese zu klein. Ich rechne immer 1% Maxleistung mal Anzahl MPPT-Kanäle. So bin ich auf 12 gekommen. Für das min Leistungslimit das selbe. So fahre ich recht gut, aber bei Waschmaschine oder Herd im Taktbetrieb kommt auch mein Aufbau an die Grenzen. Gefühlt wird es besser, wenn ich ne größere ohmsche Last zuschalte. Z.B. 50Watt alte Halogenbeleuchtung.
Zweite Vermutung: Der Shelly EM. Meiner "dreht manchmal durch". Ich ändere nichts im Haushalt, aber es kommt ein fetter peak und zwingt den DPL in eine unnötige Regelung. Ich lerne jetzt schon C++, möchte einen Filter für den Stromzähler programmieren. Wir haben beim Stromzähler eine Unterabtastung der Meßgröße und dadurch Aliasing-effekte. Vielleicht habt ihr sowas schon mal versucht und wieder verworfen? Wäre hilfreich, bevor ich in eine Sackgasse renne.
Danke und bin gespannt, ob das hilft, MFG!

**Comment by SW-Niko** [comment]
Hallo @hoschiking ,
das ist wirklich komisch. Bei einem Test über Nacht sollte die Last jede Nacht nahezu identisch sein. Hast du noch andere Daten außer dem Netzbezug? Und gibt es einen speziellen Grund warum du Eingang 2 für die Regelung verwendest?

**Reply by hoschiking** [reply]
Ja, die Last ist nachts nahezu identisch, das einzige was halbwegs dynamisch ist, ist der Kühlschrank, der Rest nur reine Standby Verbraucher. 
Ich zeichne alle möglichen Daten auf, finde jedoch nichts ungewöhnliches. Was ich mittlerweile in den Daten sehe, dass der hm600 bei Lasten<100 Watt sich sehr stark schwingt, selbst bei manuell eingestellter statischer Leistung. Darüber hinaus beruhigt sich der. Beim hm300 ist das Verhalten ähnlich, jedoch weiter weniger stark ausgeprägt und dort selten störend. Das sieht man auch in den Daten. Ich habe für Testzwecke Kanal 2 ausgesucht, die Nacht davor war Kanal 1 eingestellt, die Änderung hatte aber keine Auswirkung. 
Ich bin jetzt 3 Wochen im Urlaub, kann daher keine weiteren Tests fahren, habe die Anlage abgeschaltet für die Zeit. Wahrscheinlich werde ich auf den hm300 zurückbauen und mich nicht mehr weiter drum kümmern müssen. Andere Projekte gehen jetzt vor, es wird wieder im Herbst interessant.

**Comment by oldman4U** [comment]
Morgen. Ich verwende für die Nulleinspeisund eine DTU PRO und den Chint Meter mit einem HM800 an 24V auf 40V upconverted und das geht von 20 bis 820W auf +/- >5W genau, nur um 100W Verbrauch schwankt es um die +/- 30W. Ich versuche diesen Bereich zu vermeiden, was nicht sehr schwer ist.

**Reply by peff74** [reply]
das entspricht auch meinen Erfahrungen mit einem HM-700.
Als ich beide Eingänge geschaltet hatte, kam es zu starken Schwankungen im Bereich 100-200W.
Mein System ist S12 also ein"44Volt".
Ich nutze den Bereich viel und fahre daher wieder nur auf einem Eingang.
Denke sobald 2 Tracker ins Spiel kommen passiert das.

**Reply by TomBrett72** [reply]
Tach! Klingt nachvollziebar. Frage @oldman4U wie hast du das mit dem stepup realisiert? Jeder Kanal einen eigenen DC-DC? Meine Idee wäre, jedem HM-Kanal eine andere Arbeitsspannung zu geben. Dann müssten sich doch die Arbeitspunkte der MPPTs voneinander unterscheiden und die gegenseitige Beeinflussung aufhören. Sozusagen unabhängige PV-Module für den HM simulieren. Die Probleme sind an sich wohl bekannt, in der WIKI steht ja auch, daß einkanalige HMs empfohlen werden. Bleibt nur die Frage, ob das mit den einfachen nicht galvanisch getrennten DC-DCs möglich ist.

**Reply by oldman4U** [reply]
Ja. Jeder einen eigenen Konverter und beide auf ca. 40V. Kann einen mal hochdrehen. Wäre 50V genug für den Test?

**Reply by peff74** [reply]
@TomBrett72 
Wo steht das denn?
`daß einkanalige HMs empfohlen werden`
Wenn ich das gewusst hätte, hätte ich mir einiges an Testerei sparen können^^

**Reply by TomBrett72** [reply]
@oldman4U 50V wäre der Bereich den unsere Freunde hier mit den 48V Batterien erfolgreich nutzen. Müsste also gehen.

@peff74 bin selber erst vor kurzem drübergestolpert als ich aus "Langeweile" einfach alle Kapitel durchgeblättert hab:
https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/HOYMILES#hoymiles-inverter-selection
hab das Projket hier vor 1 Jahr auch noch nicht gekannt und hab somit die PV-Anlage mit 2 Modulen plus HM-600 bestellt... Willkommen im Club!

**Reply by peff74** [reply]
@TomBrett72 ah interessant. Ich hatte selbst ein Thema dazu eröffnet. #983
Da mein HM700 noch deutlich mehr schwankt als ein HM600 mit beiden Kanälen parallel.
Hatte beides mal getestet und bin zum entschluss gekommen bei einem Kanal HM700 vorerst zu bleiben.
Da wäre dann ja die Frage welche Hoymiles alle in Frage kommen mit einem Tracker.

Ggf. könnte man dann mit einem HM1200 bis 600W sauber regeln. Da hier nur zwei Tracker drin sind.
Wäre interessant zu wissen.

**Reply by TomBrett72** [reply]
@peff74 ja, Du hast auch das #808 eröffnet, da sind wir uns schon mal "begegnet". Vielleicht ist das alles die gleiche Ursache. Auch meine rätselhaften Leistungswerte im logfile. Ich hab die Schwankungen nur mit erhöhter Hysterese (12) in den Griff bekommen. Aber manchmal kommen eben peaks "aus dem Nichts".

**Reply by peff74** [reply]
@TomBrett72 
Ah! Das lag aber bei mir wahrscheinlich an verzögerten Meldungen des Stromzählers

**Reply by oldman4U** [reply]
Hab jetzt einen Konverter auf 50V gestellt. Gefühlt gleiches Ergebnis bei 100W Verbrauch. Ich stelle derzeit auf 16s 50V um, wird aber noch 2-3 Monate dauern bis die zweite Batterie da ist. Werde dann noch einmal testen und berichte dann.

**Reply by TomBrett72** [reply]
@oldman4U Vielen Dank! Das ist sehr interessant! Ich würde jetzt schlußfolgern, daß es doch eher an der galvanischen Kopplung der 2 MMPTs liegt und nicht unbedingt an der Eingangsspannung. Mein HM-600 war völlig unauffällig als er an jedem Eingang ein PV-Modul hatte. Mit einer zweiten Batterie könnte man diesen diskreten Aufbau auch testen. Das als PV-System aufzubauen wird aber unpraktikabel. Vielleicht läuft mir ein galvanisch trennender DC-DC über den Weg...

**Reply by oldman4U** [reply]
Gerne. Nur weil ich es vorher nicht erwähnt habe. Jeder Upconverter hängt an einer eigenen 24V 200ah Batterie.

**Reply by TomBrett72** [reply]
@oldman4U WOW! Da hast Du ja n echt starkes System und somit auch schon belegt, daß selbst 1 Batterie pro MPPT nichts bringt.

**Reply by oldman4U** [reply]
Leider gibt es da offensichtlich keinen Workaround. Werde noch einen aktuellen Wechselrichter besorgen. Ein kurzer Test mit einem HMS800 hat zumindest einen Unterschied aufgezeigt. So konnte ich bei gleicher Batterie bis ca. 80% Leistung hoch bevor nichts mehr ging, statt bis 50%. Vielleicht ist der auch um die 100W stabiler.

**Reply by oldman4U** [reply]
Würde ein Test mit 2xHM-400 statt einem 800er sinnvoll sein?

**Reply by peff74** [reply]
@oldman4U
genau das habe ich mich auch schon gefragt^^ 
Wollte je einen Kanal des HM600 und des HM700 dranhängen, fehlte aber bisher die Zeit.
Lese gerade Du willst auf 16s umstellen? Mögen die Hoymiles nicht, der Wirkungsgrad sinkt!
Max. ist 12S damit die Maschinen im optimalen Bereich bleiben.
Da gab es hinreichende Tests zu :-)

**Reply by TomBrett72** [reply]
Moin! Coole Idee! 2 HM's haben vielleicht das Problem der Kopplung nicht. Jedoch... wie regelt man die Kisten? Wenn man 200W Dauergrundlast hätte, könnte man einen HM konstant laufen lassen und den zweiten regeln. Gab irgendeinen thread, wo das behandelt wurde.

**Reply by peff74** [reply]
Verdammt, auch alles Mist.
Dann könnte man ggf. einen HM1200 oder 1500 testen, die haben ja nur 2 Tracker aber 4 Eingänge.
Wie in dem Link von Dir beschrieben.
Dann könnte man 2 nutzen und käme bis 600W bzw. 750W, was für kleine Anlagen völlig ausreicht.

Denn Träum davon, dass man E-Herd oder Waschmaschine damit betreiben kann... habe ich schon lange angehakt.
Da kommt die Regelung nicht hinterher, sondern läuft hinterher. Die Folge Strom geht in Netz und selbst zieht man fleißig.

Ich laufe wieder auf einem Eingang des HM700 und dass sieht absolut super aus!
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/91614910/7bcb68d5-e828-4a69-8bc8-b688b0eee2d1)
Egal bei welcher Auslastung.

**Reply by oldman4U** [reply]
@peff74 Wo kann ich das nachlesen?

Ich habe es noch nie versucht aber es sollte möglich sein 2 oder mehr Wechselrichter in der Hoymiles Nulleinspeisung zu verwenden.

**Reply by peff74** [reply]
Oh, das ist irgendwo erwähnt, dass der DPL aktuell nur einen Hoymiles kann.

**Reply by oldman4U** [reply]
Sorry. Meinte das mit 16s.

**Reply by peff74** [reply]
Auch 16s is zu viel.
Die sogenannte Spitzenleistung des MPPT liegt bei 29~48V.
Also nicht mehr als 12S. Vielleicht noch 13S, is aber schlecht zu bauen.
Denn schon ab 47V sinkt der Wirkungsgrad spürbar.
Es gab dazu ausführliche Tests im AkkuDoctor Forum.

**Reply by TomBrett72** [reply]
Also, den Eindruck, welchen ich hier im Projekt-Forum erlangt hab: 24V wird nicht empfohlen, die Leut mit den XXL-Anlagen fahren >48V. Ja, das ist über dem Arbeitsbereich des MPPT aber unter 60V, was die HM-Eingänge verkraften. Das wäre jetzt für mich noch ein Grund, warum es in dieser Konfiguration keine Regelschwankungen mehr gibt. Der MPPT ist schlicht inaktiv. Könnte @oldman4U mit dem stepup ja auch mal testen, ob 2 mal 50V Ruhe geben. DAS wäre ja dann DER workaround :)
@peff74 Zur Frage ob 600W aus einem HM1200 rauskommen. Wahrscheinlich auch nicht, wenn auch dort die 24V-Problematik mit den hohen Strömen am shunt zuschlägt. Und ja, Wasser heiß machen mit größer 2000W wird für ein Balkonkraftwerk schwierig.  Aber, hab derzeit eine 120Watt-Begrenzung im DPL aktiv und setzte damit ~1,5kWh am Tag um. Mehr Sonne fangen meine 2 Balkonmodule a 410Wp derzeit eh nicht ein. @oldman4U ich rechne gerade wieviel Module man braucht, um 2x 24V x200Ah überhaupt voll zubekommen...

**Reply by oldman4U** [reply]
Deshalb plane ich irgendwann auf den HMS umzusteigen, der geht dann offiziell bis 60V und scheint auch sonst noch eine gute Weiterentwicklung zu sein.

Ich teste mal 50V an beiden und lass es euch wissen ob dann bei 100W Ruhe einkehrt.

In meiner Anlage werken 6x 400 Watt und 2x 430 W Module, die gehen auf Victron MPPT's, von dort in 2x 24V 200AH und da hängt jeder an einem HM800 Eingang. Damit produziere ich bis auf die Spitzen über 800W und den Zeiten wo mir der Strom geschenkt wird (und der WR daher aus ist) seit 30 März den kompletten Strom für unser Haus, das sind ca. 8kWh pro Tag. Hab mir auch extra eine kleine Klima zugelegt, mit der ich in der Übergangszeit und am Morgen geheizt habe, und mit der ich unter Tags die Schlafräume im Sommer kühlen werde. Denke viele gehen in diese Richtung und weil es so gut geht, habe ich Budget von meinem Chef bekommen um auf 20kWh 50V upzudaten, und das mache ich jetzt schrittweise.;-)

**Reply by peff74** [reply]
ob das bei den HMS auch so ist... kann ich nicht sagen. Da steht nichts in den Datenblättern.
Beim HMXXX steht es deutlich drin.

![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/91614910/375ab20d-11d5-471c-b7a8-764264ca1bd2)

**Reply by oldman4U** [reply]
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/54359396/f143debd-9b89-456b-a7f7-db152a20b339)

**Reply by peff74** [reply]
Exakt!
Hoymiles hat es bei der neuen Serie nicht angegeben.
Aber technisch hat jeder Regelung einen Bereich, in dem sie besonders gut funktioniert, also hoch effizient.
Und das ist niemals am oben oder unteren Ende.

**Reply by oldman4U** [reply]
Das ist klar.

Das ist ein HM400 bei Volllast (hat man zumindest so geschrieben wie ich noch jung war) an der 16s LiFePO4 bei ca. 53,4V. Auch am 800er bei 40V habe ich zwischen 95 und 96#. Habt ihr da bessere Werte?

![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/54359396/1bad582c-a042-40a9-922d-cda274a70584)

**Reply by oldman4U** [reply]
@hoschiking 

Hoffe wir schweifen nicht allzu sehr vom Thema ab.

**Reply by oldman4U** [reply]
Sorry wegen der späten Antwort. Hab jetzt beide upconverter auf fast 50V gestellt. Leider keine Änderung bei 100W Verbrauch. Hab dann noch meinen HM400 an der 16s getestet und das scheint bei 100W stabil zu sein.

**Comment by 1-am-r00t** [comment]
Ist es nicht mittlerweile bekannt, dass 24V Akkusysteme gerne zum Schwingen neigen verglichen zu 48V Systemen? Kommt bei mir auch immer wieder mal vor bei meinem 24V System.

**Reply by oldman4U** [reply]
Meiner Erfahrung nach entsteht das Schwimmen wenn der WR nicht die ihm zugedachte Leistung bringt. Das ist bei 24V und den HM- ab ca. 50% und der HMS-800 den ich kurz getestet habe war bis ca. 80% stabil.

**Reply by hoschiking** [reply]
Soweit ich es mitbekommen habe, sind die 48V Systeme genauso davon betroffen, thereotisch müßte es sogar sich verschlimmern,
genauso wie bei 24V System mit nur einem String zu 2 Strings., da sich jede Verschiebung des Arbeitspunktes des MPP-Trackers doppelt so stark auswirkt. Mangels 48V Speicher kann ich es jedoch nicht testen.

**Comment by hoschiking** [comment]
Ich habe meine Anlage auf den HM-300 zurückgebaut und das gewohnte Verhalten hat sich wieder eingestellt.
Oberes Diagramm die tatsächliche Leistung des HM-300:
![Screenshot_6](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/159241718/e55862c3-40ae-4418-b0b5-a8afb666072b)
 
Ja auch der schwingt ab und an, ich habe aber auch bisher keine Zeit gehabt mich dessen anzunehmen und zu optimieren.
Ich vermute das Problem bei der Art der Auslesung des Stromzählers. Ich lesen den mit einem Optokopf aus und der liefert mir einen Wert nur alle 5 Sekunden. Vielleicht baue ich irgendwann einen Shelly ein, der soll angeblich fast in Echtzeit die Daten liefern.
Hat hier jemand mit einem Shelly 3EM (mit und ohne pro) Erfahrungen gesammelt und kann was dazu berichten?

**Reply by TomBrett72** [reply]
Hi. Meine Whg hat nur 1 Phase, deshalb komme ich mit dem Shelly EM aus. Also die "ohne Pro Serie". Ich hab die Verbindungen zwischen DTU und Shelly in der Fritz!Box "echtzeitpriorisiert" und konnte dann mit dem timeout auf 500ms runtergehen. Es tut besser mit meinem HM600, manche Tage sind ohne Abschaltung. Ich denke jedoch die größte Hürde ist auch hier die Strom/Leistungsmessung im Hoymiles. Je nach Lastpunkt hab ich bis zu 20Watt Unterschied zwischen gesetztem Limit und tatsächlicher Inverterleistung. So wie ich bis jetzt den DPL verstanden habe, löst das sofort den nächsten Regelvorgang aus. Deshalb hatte ich an verschiedenen Stellen hier im Forum die Meinung vertreten, daß man die Hysterese erhöhen sollte. Ich hab 12W, 1% für jeden MPPT. Bin für besseres Wissen aber immer offen!

**Reply by hoschiking** [reply]
Danke dir für die Antwort. Alles unter 1 Sekunde klingt sehr gut. 
Kleine Anmerkung: Den 3EM ohne pro gibt es auch dreiphasig. Der hat wohl nicht alle Features und ist älter. 
Ich denke nicht, dass das erhöhen der Hysterese Einfluss auf das Schwingen hat, wenn es sich in Bereich von 20-50 Watt bewegt. Um das zu unterbinden, müsste die Hysterese auf > 50 Watt eingestellt werden.

**Comment by Artikel14** [comment]
Bei meinen Messungen mit fest eingestellter Nachtleistung war immer wieder aufgefallen, 
dass der HM-600 unter 100 Watt nicht genau genug jene Leistung einspeist, die eingestellt ist.
Beispiel: Um 62 Watt einzuspeisen, musste ich 74 Watt einstellen.
Das Verhalten war aber dann immerhin stabil und sehr reproduzierbar.

Denkbar wäre, in der openDTU eine WR-spezifische "Umsetzungstabelle" abzuspeichern,
welche die Fehlanpassung korrigiert und die Sollwerte der Einspeisung in
jene Steuerwerte umsetzt, die dem Ziel am besten nah kommen.
Man könnte die Tabelle sogar durchaus automatisch befüllen lassen.

Mit meiner manuellen Nachsteuerung bin ich sehr nah an die Nulleinspeisung gekommen.
Die Umsetzungstabelle würde, so vermute ich auch das Schwingen sehr dämpfen und
dann auch zu einer deutlichen Verkleinerung der Hysterese führe, die dann sinnvoller  nutzbar wäre, was die Schwingung weitgehend abstellen könnte.

Außerdem habe ich in 9 Monaten (Sommer, Frühjahr, Herbst) meist Überschüsse im Akku.
Dann stelle ich den Parameter "Angestrebter Netzbezug" auf einen leicht negativen Wert,
um sicherzustellen, dass eher minimal eingespeist wird, als dass es zum ungewollten kostenpflichtigen Netzbezug kommt.
In den drei Wintermonaten sind die meist knappen Energieinhalte im Akku für mich wirtschaftlich
wertvoller. Dann stelle ich den "Angestrebten Netzbezug" entsprechend auf leicht positive Werte,
um beim Überschwingen der Regelung möglichst keine wertvolle Winter-Solarenergie ungenutzt ins Netz einzuspeisen. Wirtschaftlich ist das jeweils eine zusätzliche Optimierung. 

Gerne würde ich diese beiden Steuerungen in der Software auf dem ESP32 automatisieren.

(1) Umsetzungstabelle zur Optimierung der genaueren Einspeiseleistung
(2) Kalendarische Steuerung des "Angestrebten Netzbezugs"

Beides hat das Ziel der Optimierung einer besonders schwingungsarmen und wirtschaftlichen Nulleinspeisung. Wer hätte an solchen Funktionen ebenfalls Interesse?

**Reply by TomBrett72** [reply]
Hallo @Artikel14. Zunächst bestätige ich Deine Beobachtungen am HM-600. Ist bei meinem auch so.
Auch verstehe ich Dein Vorgehen bzgl. "Angestrebten Netzbezug". Würde ich ebenfalls so machen, jedoch ist die Grundlast bei mir oft unter 20W und ich hätte dann Abschaltungen des HM-600 wegen Unterschreitung der Minimallast.
zu (1) ich kenne diese Art der Umsetzung von Kalibrierkuven für Sensoren. Zwischen den Stützpunkten, die der Nutzer erstmal sinnvoll applizieren darf, muß die Maschine dann interpolieren. Und das alles dann in den DPL... stell ich mir aufwändig vor. Müssen die Entwickler beurteilen. Da wären wir jetzt an dem Punkt, wo man auch über die Arbeitsweise des DPL diskutieren könnte. So wie ich den DPL verstanden habe, nutzt man die Stellungsrückmeldung des WR um die tatsächliche Leistung des Haushalts zu bestimmen. Man hat wohl auch schon erkannt, daß dies nicht die optimale Lösung ist. Irgendwo hab ich gelesen, daß man die echte Leistung vom HM über einen zweiten shelly an der Einspeisesteckdose ermitteln möchte. Würde in meinen Augen funktionieren. Mir schwebt prinzipiell ein anderes Verfahren vor. Mein Ansatz: Es ist doch egal, was der WR liefert! Unsere Regelgröße ist der "Angestrebte Netzbezug". Hab ich eine Abweichung von z.B. +30W muß man auf den alten Stellwert vom HM-600 nur 5% draufpacken. Dazu gibt es dann Regelgrenzen für min und max. Wahrscheinlich ist das auch nicht neu und bringt Schwierigkeiten, die ich jetzt nicht im Blick hab. Der Regelkreis muß z.B. abwarten, bis der Stellwert wirklich übertragen wurde. Sonst schwingt es definitiv.
zu (2) hm, ganz ehrlich, 2 mal im Jahr in die Konfiguration gehen wäre doch kein Aufwand. Sagst aber selbst, hängt vom Überschuss ab. Bei dem Mistwetter gerade würde ich schon auf "sparsam" schalten. Also bräuchte man da meiner Meinung nach eine andere Bezugsgröße. Vielleicht SOC>50%?
Mal sehen, ob hier noch jemand schreibt, vielleicht sollte man das als extra Punkt im "issues-enhancement" deklarieren.

**Reply by oldman4U** [reply]
Ganz verstehe ich die Problematik nicht. Es gibt einen Verbrauch X. Die DTU sagt dem Wechselrichter X zu produzieren. Wenn X danach nicht stimmt muss wieder geregelt werden. Ob dieses "nicht stimmen" durch einen inzwischen veränderten Verbrauch oder dadurch entsteht weil der WR nicht exakt den Wert X produziert ist doch irrelevant. Die Hoymileslösung ist meistens auf ca. 10 - 15W genau nach einer großen Änderung und braucht ein paar Sekunden mehr um auf die 5W genau zu regeln. Je länger sich der Verbrauch nicht stark ändert, desto weniger schwingt der Bezug/die Einspeisung.
